datamol/conformers/_conformers.py

```diff
@@ -81,11 +83,14 @@ def generate(
             are kept. This step is done after an eventual minimization step.
         clear_existing: Whether to overwrite existing conformers for the molecule.
         align_conformers: Whether to align the conformers.
-        minimize_energy: Wether to minimize conformer's energies using UFF.
+        minimize_energy: Wether to minimize conformer's energies using MMFF94s.
```

Comment:
correct typo here. `Whether`

Reply:
Agreed. UFF restored as default, 2 additional MMFF94s forcefields, one with and one without electrostatic component, offered as options.  Typo corrected.